### ISSUE TITLE
Invalid syntax 

### ISSUE THREAD

**Issue by claudio-petrini** [crazy-max/ghaction-docker-buildx#170]
Hi, first of all thanks for this great Github action.
I'm using it to build a Docker image on different architectures and since yesterday (Apr 29) everything run smoothly.

With today's builds I'm having the following error:
```
invalid argument "--push" for "--no-cache" flag: strconv.ParseBool: parsing "--push": invalid syntax
```

The command is the following:
```
docker buildx build --platform linux/ppc64le,linux/s390x,linux/arm/v7,linux/arm64 -t instathings/gate:staging --file ./Dockerfile --no-cache --push ./
```

Can you help me on this?
Thank you 

Claudio

**Comment by crazy-max** [comment]
Hi @claudio-petrini, do you have a link to your repo please?

**Comment by claudio-petrini** [comment]
Yep, [repo link](https://github.com/Instathings/gate).
By the way I've pushed on `development` by specifying the buildx version to `v0.3.0` and it works fine.
File is `build.yml`

**Comment by crazy-max** [comment]
Looks like an upstream issue: docker/buildx#265

**Comment by claudio-petrini** [comment]
Thanks for your fast response, will stay on the `v0.3.0` so far.

**Comment by crazy-max** [comment]
Or you can set `--push true ./` instead of `--push ./` (parsing issue)

**Comment by claudio-petrini** [comment]
Just tried `--push true ./` but it returns the same error.

**Comment by crazy-max** [comment]
@claudio-petrini Ok will wait for a fix on their side ;)

**Comment by crazy-max** [comment]
Fixed docker/buildx#268
